suite-etl/static/contracts.py

```
@@ -0,0 +1,217 @@
+    Args:
+        chain_id: Chain ID to filter by, or None to get all contracts
+
+    Returns:
+        dict: Dictionary of contract addresses to contract info
+    """
+    if chain_id is not None:
+        return KNOWN_CONTRACTS.get(chain_id, {})
+
+    # Combine all contracts across chains
+    all_contracts = {}
+    for chain_id, contracts in KNOWN_CONTRACTS.items():
```

Pylint (3.3.7)

[refactor] Redefining argument with the local name 'chain_id'

(R1704)